recipes-tag/regolith/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "regolith" %}
-{% set version = "0.2.0" %}
-{% set sha256 = "d278f44373d27abc858ded0e6f06b7dc35189dbe737f87025703b0982ae10a20" %}
+{% set version = "0.4.0" %}
+{% set sha256 = "3b401670ed958ce2bb63fd3584ff472baae8bda1f00d2689cc1091adaf0ca1b5" %}
```

Comment:
Is this package needed at all? I don't see any dependency on it in this PR.

Reply:
I could remove the recipe completely